internal/bminventory/inventory_test.go

```diff
@@ -10029,6 +10029,111 @@ var _ = Describe("TestRegisterCluster", func() {
 			}
 		}
 
+		// Testing the default network type setting during register flow, with various combinations of platforms,
+		// OpenShift version and high availability modes
+		// Note: clusters with high availability mode "None" must not have have Platform "baremetal", and OpenShift version >= 4.8
```

Comment:
Where is that sentence coming from? I agree that SNO is only OCP 4.8+, but nothing stops OCP 4.13 to support SNO with platform=baremetal

Reply:
SNO must be platform "none"